### PR TITLE
Add a checkbox to agree to privacy policy.

### PR DESCRIPTION
**Pull request by mrstegeman**
References #2614

### CODE REVIEW

**Review by benfrancis — COMMENTED** [review]
Let's hold off on this one for a moment while I figure out whether we also need a Terms of Service. Ideally we could just add "and Terms of Service"  to the string, but that may not be GDPR compliant.

File: static/fluent/en-US/main.ftl [hunk 1]
```diff
@@ -534,7 +534,7 @@ tunnel-setup-choose-address = Choose a secure web address for your gateway:
 tunnel-setup-input-subdomain =
     .placeholder = subdomain
 tunnel-setup-email-opt-in = Keep me updated with news about WebThings.
-tunnel-setup-privacy-policy = Privacy Policy
+tunnel-setup-agree-privacy-policy = Agree to the <a href="https://webthings.io/privacy">WebThings privacy policy</a>
```
Comment (benfrancis) [inline static/fluent/en-US/main.ftl:537]: `Agree to the WebThings <a href="https://webthings.io/privacy">Privacy Policy</a> and <a href="https://webthings.io/legal">Terms of Service</a>.`

**Review by benfrancis — CHANGES_REQUESTED** [review]
OK, my conclusion is that it's OK to agree to the Privacy Policy and Terms of Service with the same checkbox, as long as the newsletter signup is separate.

File: static/fluent/en-US/main.ftl (hunk 1 above)
Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: Updated.

**Comment by mrstegeman** [comment]
@benfrancis is this good to go now?

File: static/css/subdomain-form.css [hunk 2]
```diff
@@ -112,20 +112,24 @@ button:active:enabled {
   background-color: #658196;
 }
 
+#agree-group,
 #opt-in-group {
   padding-top: 2rem;
```
Comment (benfrancis) [inline static/css/subdomain-form.css:117]: I suggest `padding-top: 0;`

File: static/fluent/en-US/main.ftl [hunk 3]
```diff
@@ -534,7 +534,7 @@ tunnel-setup-choose-address = Choose a secure web address for your gateway:
 tunnel-setup-input-subdomain =
     .placeholder = subdomain
 tunnel-setup-email-opt-in = Keep me updated with news about WebThings.
-tunnel-setup-privacy-policy = Privacy Policy
+tunnel-setup-agree-privacy-policy = Agree to the WebThings <a href="https://webthings.io/privacy">Privacy Policy</a> and <a href="https://webthings.io/legal">Terms of Service</a>.
```
Comment (benfrancis) [inline static/fluent/en-US/main.ftl:537]: Unfortunately the hyperlinks are not rendering for me. I see a fluent error in the logs...
`An element of forbidden type "a" was found in the translation. Only safe text-level elements and elements with data-l10n-name are allowed.`

**Review by benfrancis — CHANGES_REQUESTED** [review]
Sorry, I missed the new review request. A couple of comments inline.

File: static/css/subdomain-form.css (hunk 2 above)
Comment (mrstegeman) [reply static/css/subdomain-form.css]: Done.

File: static/fluent/en-US/main.ftl (hunk 3 above)
Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: Fixed.

**Review by benfrancis — APPROVED** [review]
Great, thanks.

**Pull request merged**